src/main/java/nl/hsac/fitnesse/fixture/slim/DateFormatFixture.java

```diff
@@ -0,0 +1,73 @@
+package nl.hsac.fitnesse.fixture.slim;
+
+import java.text.ParseException;
+import java.text.SimpleDateFormat;
+import java.util.Date;
+import java.util.TimeZone;
+
+/**
+ * Fixture that helps converting dates to and from various formats. Intended to be used as a utility fixture i.e. as library
+ */
+
+public class DateFormatFixture extends SlimFixture {
+    private String dateFormat = "dd-MM-yyyy";
```

Comment:
Why not default to `((SimpleDateFormat) DateFormat.getDateInstance(DateFormat.SHORT)).toPattern();`, which uses the Java runtime default locale. For the Netherlands that would get you 'd-M-yy', which means no leading zeroes and year in two digits.
In tests you could/should than explicitly state the format to use, since that is probably not what you want exactly, but it is nicer for users in countries using a different format.